criu/include/cr_options.h

```diff
@@ -157,6 +157,7 @@ struct cr_options {
 	int track_mem;
 	char *img_parent;
 	int auto_dedup;
+	int zero_pages;
```

Comment:
Would it be possible to add support for this option in [cr-service](https://github.com/checkpoint-restore/criu/blob/cda1c5c95aaca5affa28daa59d7f25ed64007a78/criu/cr-service.c#L288) and [libcriu](https://github.com/checkpoint-restore/criu/blob/criu-dev/lib/c/criu.c)?

Reply:
I tried to do it but I must confess that I'm not familiar with this stuff so please take a second look and let me know if there's anything left?